api/admin/controller/patron_auth_services.py

```diff
@@ -16,6 +16,9 @@
 from api.saml.provider import SAMLWebSSOAuthenticationProvider
 from api.simple_authentication import SimpleAuthenticationProvider
 from api.sip import SIP2AuthenticationProvider
+from api.sirsidynix_authentication_provider import (
+    SirsiDynixHorizonAuthenticationProvider,
```

Comment:
We will be using this integration with both Horizon and Symphony, so it probably makes sense to just call this `SirsiDynixAuthenticationProvider` unless we end up needing two different implementations, one for Horizon and one for Symphony.

Reply:
Unless we want to be fancy and have some dynamism in the class based on what API we're hitting. Which seems unnecessarily complex.

Reply:
Two implementations is fine with me. They hopefully can share some code, since I think they will be fairly similar. Since we need support for both Horizon and Symphony we can either do this in this PR, or add an additional ticket for supporting the other.

Reply:
We can do it in this ticket in another ticket, we would need access to a symphony environment though. The current API endpoint is Horizon.